src/app/services/project.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+
+import { Project } from '../models/project';
+
+@Injectable()
+export class ProjectService {
+
+    private storageKey: string = 'cclab.projects';
+    private projects: Project[] = [];
+    private storage: Storage = window.localStorage;
```

Comment:
Not sure how `window` is supposed to be handled in Angular 2, but seems like we probably shouldn't be referencing it directly here. Should it be [injected](http://stackoverflow.com/questions/34177221/angular2-how-to-inject-window-into-an-angular2-service) instead? How could we mock this for test?